$MirOS: ports/lang/python/2.3/patches/patch-Modules_datetimemodule_c,v 1.1 2007/07/19 20:26:15 bsiegert Exp $
--- Modules/datetimemodule.c.orig	2008-02-14 12:26:18.000000000 +0100
+++ Modules/datetimemodule.c	2008-04-22 23:06:09.000000000 +0200
@@ -2259,7 +2259,7 @@ date_local_from_time_t(PyObject *cls, do
 	tm = localtime(&t);
 	if (tm)
 		result = PyObject_CallFunction(cls, "iii",
-					       tm->tm_year + 1900,
+					       (int)(tm->tm_year + 1900),
 					       tm->tm_mon + 1,
 					       tm->tm_mday);
 	else
